payment-request/payment-request-show-accept-promise-returned.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE HTML>
+<meta charset='utf-8'>
+<title>PaymentRequest: when calling show() the acceptPromise should get created and returned.</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<p></p>
+
+<script>
+async_test((t) => {
+  const paymentArgs = [[{supportedMethods: ['non-supported-payment-method']}], {total: {label: 'label', amount: {currency: 'USD', value: '5.00'}}}];
+
+  onload = t.step_func_done(() => {
+    let pr = new PaymentRequest(paymentArgs)
```

Comment:
The`PaymentRequest` constructor requires at least two separate arguments: a _methodData_ argument and a _details_ argument, but this is feeding it a single array as an argument.